src/pages/index.js

```diff
@@ -5,29 +5,65 @@ import UserInfo from "../components/UserInfo.js";
 import Section from "../components/Section.js";
 import PopupWithImage from "../components/PopupWithImage.js";
 import "./index.css";
-import { initialCards, settings } from "../utils.js";
+import { settings } from "../utils.js";
+import Api from "../components/Api.js";
 
 // WRAPPERS
-const cardsWrap = document.querySelector(".cards__list");
 const profileEditModal = document.querySelector("#profile-edit-modal");
 const addCardModal = document.querySelector("#add-card-modal");
-const previewImageModal = document.querySelector("#preview-image-modal");
+const editAvatarModal = document.querySelector("#edit-avatar-modal");
 const profileEditForm = profileEditModal.querySelector(".modal__form");
 const addCardFormElement = addCardModal.querySelector(".modal__form");
+const editAvatarForm = editAvatarModal.querySelector(".modal__form");
+
+// Import from Api
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "3e30a101-1104-4b9d-af70-47714cfc57be",
+    "Content-Type": "application/json",
+  },
+});
+
+// Fetch and use user info
+api
+  .getUserInfo()
+  .then((userData) => {
+    userInfo.setUserInfo({ name: userData.name, job: userData.about });
+    document.querySelector(".profile__image").src = userData.avatar;
```

Comment:
My remark hasn't been corrected https://github.com/chriskapano/se_project_aroundtheus/pull/1#discussion_r1810097966 

`  document.querySelector(".profile__image").src = userData.avatar;
`

`UserInfo` should set the entire profile data (including the **avatar**)


```js

  .then((userData) => {
    userInfo.setUserInfo({
      name: userData.name,
      job: userData.about,
      avatar: userData.avatar,
    });
  })
  .catch((err) => {
```